accounts/views.py

```
@@ -20,65 +20,68 @@
             audio_record__in=AudioRecord.objects.filter(
-                subject__phone_number=request.session['subject_login']
+                subject__phone_number=request.session["subject_login"]
             )
-        )
+        ),
     }
 
     return render(request, "home.html", context)
 
 
 def logout(request):
```

Ruff (0.12.2)

Redefinition of unused `logout` from line 46

(F811)